lib/attr_encrypted.rb

```diff
@@ -277,6 +310,7 @@ module InstanceMethods
     #  @user = User.new('some-secret-key')
     #  @user.decrypt(:email, 'SOME_ENCRYPTED_EMAIL_STRING')
     def decrypt(attribute, encrypted_value)
+      self.class.encrypted_attributes[attribute.to_sym][:operation] = :decrypting
```

Comment:
I initially thought there would be more time between setting the "shared options" and then making a private copy of them for the actual encrypt/decrypt call, but when I read farther down I saw that it was pretty much one right after the other. I don't think it should have any multithreading issues.

Regarding the new commit, I like that better :+1: